sysbuild/Kconfig.secureboot

```
@@ -62,68 +62,66 @@ config SECURE_BOOT_SIGNING_KEY_FILE
-	  Specifies the private key used for signing the firmware image.
-	  The hash of the corresponding public key is stored as the first
-	  entry in the list of public key hashes in the provision hex file.
-	  This value can also be set by exporting an environment variable
-	  named 'SECURE_BOOT_SIGNING_KEY_FILE' or passing
-	  '-DSECURE_BOOT_SIGNING_KEY_FILE=/path/to/my/pem' when running cmake.
+	  Absolute (or relative, to the application's APPLICATION_CONFIG_DIR) path to the private
+	  key PEM file.
+
+	  Specifies the private key used for signing the firmware image. The hash of the
+	  corresponding public key is stored as the first entry in the list of public key hashes
+	  in the provision hex file.
+
 	  See also SECURE_BOOT_PUBLIC_KEY_FILES.
 
 config SECURE_BOOT_SIGNING_COMMAND
 	string
 	prompt "Custom signing command" if SECURE_BOOT_SIGNING_CUSTOM
 	default ""
 	help
-	  This command will be called to produce a signature of the firmware.
-	  It will be called as "${CONFIG_SECURE_BOOT_SIGNING_COMMAND} <file>"
-	  The command must calculate the signature over the contents
-	  of the <file> and write the signature to stdout.
-	  The signature must be on DER format.
+	  This command will be called to produce a signature of the firmware. It will be called as:
+	  "${SB_CONFIG_SECURE_BOOT_SIGNING_COMMAND} <file>"
+
+	  The command must calculate the signature over the contents of the <file> and write the
+	  signature to stdout. The signature must be in DER format.
 
 config SECURE_BOOT_SIGNING_PUBLIC_KEY
 	string
 	prompt "Public key PEM file" if SECURE_BOOT_SIGNING_CUSTOM
 	default ""
 	help
 	  Path to a PEM file.
-	  When using a custom signing command, specify the corresponding public
-	  key here. This public key is checked during building, and added as
-	  the first entry in the provisioned data. See SECURE_BOOT_PUBLIC_KEY_FILES.
+	  When using a custom signing command, specify the corresponding public key here. This
+	  public key is checked during building, and added as the first entry in the provisioned
+	  data. See SECURE_BOOT_PUBLIC_KEY_FILES.
 
 config SECURE_BOOT_PUBLIC_KEY_FILES
 	string "Public Key PEM files"
 	default ""
 	help
-	  Comma-separated list of absolute paths to public key pem files.
-	  The provision hex file will contain a list of hashes of public keys.
-	  The first public key hash is the one corresponding to the private
-	  signing key used to sign the image. See SECURE_BOOT_SIGNING_KEY_FILE.
-	  The hashes of the public keys specified in this configuration will be
-	  placed after the aforementioned public key hash, in the order
-	  they appear in this config. The order is significant since if an image
-	  is successfully validated against a public key in the list, all
-	  public keys before it in the list will be invalidated.
+	  Comma-separated list of absolute (or relative, to the application's
+	  APPLICATION_CONFIG_DIR) paths to public key pem files. The provision hex file will
```

Currently not approved.

See https://github.com/nrfconnect/sdk-nrf/pull/16124/files#r1650921643 for details.

---

Conversation was marked resolved, but no changes seems to have been pushed ?

---

Why would changes be pushed? The text is accurate and linked PR is merged

---

> Why would changes be pushed? The text is accurate and linked PR is merged

because text says:
> or relative, to the application's APPLICATION_CONFIG_DIR

which is not precise. In most cases it's correct that the application's and sysbuild's APPLICATION_CONFIG_DIR will be identical, but technically it's relative to sysbuild's APPLICATION_CONFIG_DIR.

Perhaps just write:
```suggestion
	  Comma-separated list of absolute (or relative, to the APPLICATION_CONFIG_DIR)
	  paths to public key pem files. The provision hex file will
```

after all, this is a sysbuild specific Kconfig file, and because APPLICATION_CONFIG_DIR is shared across sysbuild and images in then normal case.
Advanced users should be aware of the difference, so not mentioning `application's` vs `sysbuild's` avoid confusing regular users.